app/services/guidance_service.rb

```diff
@@ -20,9 +20,22 @@ def orgs
     end
     return @orgs
   end
+
   def any?(org:nil, question:nil)
-    if org.nil? && question.nil?
-      return hashified_annotations? || hashified_guidance_groups?
+    if org.nil?
+      if question.nil?
+        return hashified_annotations? || hashified_guidance_groups?
+      end
+      if question.present?
+        # check each annotation/guidance group for a response to this question
+        # Would be nice not to have to crawl the entire list each time we want to know this
+        anno = orgs.reduce(false) {|found, o| found || guidance_annotations?(org: o, question: question)}
```

Comment:
agreed, there's likely a more efficient way to check this. Maybe a query in the other direction passing in the list of org_ids and using `joins` which uses an `inner join` so would return an empty set for us
We can tackle this later as an enhancement or if you have time during this sprint feel free to send a PR. I'm going to push this one through now so the visual stuff can be UATed.